File: src/test/java/murphy/software/games/platform2d/move/config/ESpeedTest.java
package murphy.software.games.platform2d.move.config;

import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertEquals;

public class ESpeedTest {

    @Test
    void testSpeed(){
        assertEquals(4.0, ESpeed.walk.getDistance());
    }
}

File: src/test/java/murphy/software/games/platform2d/move/config/ESpeedTest.kt
package murphy.software.games.platform2d.move.config

import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test

class ESpeedTest {

    @Test
    fun testSpeed() {
        Assertions.assertEquals(6.0, ESpeed.walk.feetPerSec)
    }
}